torch/_dynamo/config.py

```diff
@@ -428,7 +428,7 @@ def default_debug_dir_root():
 only_allow_pt2_compliant_ops = False
 
 # This flag is ignored and maintained for backwards compatibility.
-capture_autograd_function = True
+capture_autograd_function = False
```

Comment:
Do you have more context behind this? If this is causing log spew then an alternative is to delete whatever is spewing logs when capture_autograd_function = True.

Reply:
Also, it looks like the codepath is actually being used, so turning it to False might be bad

Reply:
@zou3519 sounds good, I'll put up another diff shortly. Out of curiosity is there a functional difference between these two approaches? Is the basic idea that many people have this flag hard coded to True in their code and thus will potentially get confused and/or have log spew?

Reply:
Well we're just trying to avoid log spew and confusion. To avoid log spew we have chosen the least likely option (False), to avoid developer confusion the proposal is that the flag's value should be True by default.